recipes-robot/srobo-firewall/srobo-firewall.bb

```diff
@@ -0,0 +1,28 @@
+SUMMARY = "Package for SR nftables configuration"
+inherit systemd
+
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=691d84ab639c4b8173db302930bf71f4"
+
+SYSTEMD_AUTO_ENABLE = "enable"
+SYTEMD_SERVICE:${PN} = "nftables.service"
```

Comment:
```suggestion
SYSTEMD_SERVICE:${PN} = "nftables.service"
```

Reply:
I assume given this typo that this has not yet been tested?